src/encoded/static/components/search.js

```diff
@@ -1099,6 +1099,7 @@ export class ResultTable extends React.Component {
         const trimmedSearchBase = searchBase.replace(/[?|&]limit=all/, '');
         let browseAllFiles = true; // True to pass all files to browser
         let browserAssembly = ''; // Assembly to pass to ResultsBrowser component
+        let browserRegion = '';  // Region that may be passed to browser
```

Comment:
I would have used `this.props.region` instead of a new variable, as nothing intense is happening with the value. Personal preference.